gems/pending/util/mount/miq_generic_mount_session.rb

```diff
@@ -63,12 +63,7 @@ def self.uri_scheme_to_class(uri)
   end
 
   def mount_share
-    # Check if settings contains the mount_point to use
-    @mnt_point = @settings[:mount_point].blank? ? File.join(self.class.base_mount_point, "miq_#{MiqUUID.new_guid}") : @settings[:mount_point]
-    raise MiqException::MountPointAlreadyExists, "#{@mnt_point} directory already exists!!!" if File.exist?(@mnt_point)
-
-    FileUtils.mkdir_p @mnt_point
-    raise MiqException::MiqLogFileMountPointMissing, "mount point: [#{@mnt_point}] failed to be created" unless File.directory?(@mnt_point)
+    @mnt_point = Dir.mktmpdir("miq_", settings_mount_point)
```

Comment:
Let's require 'tmpdir' for Dir.tmpdir and Dir.mktmpdir before we use them.
I'm wary of relying on something else requiring it for us.

```ruby
irb(main):001:0> Dir.tmpdir
NoMethodError: undefined method `tmpdir' for Dir:Class
	from (irb):1
	from /Users/joerafaniello/.rubies/ruby-2.2.3/bin/irb:11:in `<main>'
irb(main):002:0> require 'tmpdir'
=> true
irb(main):003:0> Dir.tmpdir
=> "/var/folders/fq/blrz820d3qz7nm7vj8mbtfs40000gq/T"
```